source/CardGames.py

```diff
@@ -1,10 +1,9 @@
-
 import random
 import os
 
 cardImages = []
-values = list(range(1,14))
-suits = ["Spades", "Clubs", "Hearts", "Diamonds"]
+values = list(range(1,43))
+sprint_value = [1, 2]
 
 def find_root_dir():
   cwd = os.getcwd()
```

Comment:
I can't comment on line 25, but I think eventually the short image should be adjusted to show overlapped cards for burn values

Reply:
Great idea Sean. If its okay for you, I can go ahead and create a separate task to implement the overlap feature. if you are okay with it otherwise, lets go ahead and get this pushed for now. Thanks again!